docs/references/go/overview.mdx

```diff
@@ -44,17 +50,19 @@ The following examples demonstrate both approaches.
 
 If you only use one API key, you can import the packages required for the APIs you need. Then you can execute your desired request methods as functions, such as `$resource$.Get()` or `$resource$.Delete()`. The following example demonstrates this process:
 
+<InjectKeys>
+
 ```go
 import (
-    "github.com/clerk/clerk-sdk-go/v2"
-    "github.com/clerk/clerk-sdk-go/v2/$resource$"
+  "github.com/clerk/clerk-sdk-go/v2"
+  "github.com/clerk/clerk-sdk-go/v2/$resource$"
 )
 
 // Each operation requires a context.Context as the first argument.
 ctx := context.Background()
 
-// Set the API key
-clerk.SetKey("sk_live_XXX")
+// Set the API key with your Clerk secret key
+clerk.SetKey("{{secret_key}}")
```

Comment:
:question: This is not working code, but rather a placeholder/example. 

Just confirming we want to use the real keys here as well.

Reply:
We try our best to format all of our code examples as working code examples, because users copy and paste them - would it be too much trouble to make these working code examples? I'm not familiar with Go, so if it is too much trouble, we can keep what you had!

Reply:
okay gotcha ! I'll remove the keys :)